pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -32,14 +32,23 @@ import (
 )
 
 // PluginName indicates name of volcano scheduler plugin.
-const PluginName = "proportion"
+const (
+	PluginName = "proportion"
+	// overCommitFactor is resource overCommit factor for enqueue action
+	// It determines the number of `pending` pods that the scheduler will tolerate
+	// when the resources of the cluster is insufficient
+	overCommitFactor = "overcommit-factor"
```

Comment:
Variables in this style are deprecated.Does not conform to k8s style.
```suggestion
	overCommitFactor = "overcommitFactor"
```